src/current/releases/cloud.md

```diff
@@ -14,6 +14,14 @@ Get future release notes emailed to you:
 
 {% include marketo.html formId=1083 %}
 
+## December 16, 2024
+
+<h3 id="2024-12-16-general-updates"> General updates </h3>
+
+- CockroachDB {{ site.data.products.cloud }} is now available as a pay-as-you-go offering on the Google Cloud Marketplace. This allows Google Cloud Platform customers to pay for CockroachDB {{ site.data.products.cloud }} charges via their Google Cloud accounts, with no up-front commitments. For more detail, refer to:
```

Comment:
I'm not sure why, but Partner Marketing advised us to use Google Cloud instead of Google Cloud "Platform". Maybe it's part of a rebrand? But we've been trying to say Google Cloud in product and marketing material.